Unity-Technologies/ScriptableRenderPipeline
# Vfx/feature/link impossible reason

### Purpose of this PR
Display an explicit message about why an edge is impossible to create in the visual effect graph.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- [n/a ] Built a player
- [ n/a] Tested UI multi-edition + Undo/Redo
- [ X] C# and shader warnings (supress shader cache to see them)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low
**Halo Effect**: None, Low, Medium, High?
Low
